okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -416,6 +501,26 @@ public OkHttpClientStream newStream(
     }
   }
 
+  private Optional<TrustManager> getX509ExtendedTrustManager(TlsChannelCredentials tlsCreds)
+      throws GeneralSecurityException {
+    TrustManager[] tm = null;
+    // Using the same way of creating TrustManager from
+    // {@link OkHttpChannelBuilder#sslSocketFactoryFrom}.
```

Comment:
Don't use Javadoc-style in regular comments. `OkHttpChannelBuilder.sslSocketFactoryFrom()`

Reply:
Done.